runtime/doc/diagnostic.txt

```diff
@@ -265,6 +265,16 @@ enable({bufnr}, {namespace})                         *vim.diagnostic.enable()*
                     {namespace}  number|nil Only enable diagnostics for the
                                  given namespace.
 
+fromlist({list})                                   *vim.diagnostic.fromlist()*
```

Comment:
- is there a reason we would not stick with the "qflist" convention? That clearly communicates what kind of list this is. Especially given that this also returns... a list.
- same for `tolist()` => `toqflist()`

```suggestion
fromqflist({list})                                   *vim.diagnostic.fromlist()*
```

Reply:
I am fine with this, hopefully it's clear that this can be use for the loclist as well though.

I prefer no `_` personally (e.g. `fromqflist` instead of `from_qflist`).

Reply:
> I am fine with this, hopefully it's clear that this can be use for the loclist as well though.

yeah, it's understood that loclists have qflist format/structure. Or we need to establish that understanding, because having "loclist" variants everywhere or just calling them "lists" is strictly worse.




> I prefer no `_` personally (e.g. `fromqflist` instead of `from_qflist`).

fine with me :)